src/ListItemSection/test/ListItemSection.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { createRendererWithUniDriver, cleanup } from '../../../test/utils/unit';
+
+import ListItemSection from '../ListItemSection';
+import { listItemSectionPrivateDriverFactory } from './ListItemSection.private.uni.driver';
+
+describe('ListItemSection', () => {
+  const renderListItemSection = (props = {}) => <ListItemSection {...props} />;
+  const render = createRendererWithUniDriver(
+    listItemSectionPrivateDriverFactory,
+  );
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('should render', async () => {
+    const { driver } = render(renderListItemSection());
+
+    expect(await driver.exists()).toBeTruthy();
+  });
+
+  it('should match title', async () => {
+    const { driver } = render(
+      renderListItemSection({ title: 'ListItemSection title' }),
+    );
+
+    expect(await (await driver.getTextDriver()).getText()).toEqual(
```

Comment:
Could be much simpler:
```suggestion
    expect(await driver.getTitle().text()).toEqual(
```